www/commands/render.md

```diff
@@ -4,6 +4,13 @@ title: render - ///_hyperscript
 
 ## The `render` command
 
+### Installing
+
+Note: if you want the template command, you must include the /dist/template.js file in addition to the hyperscript script
+  ~~~ html
```

Comment:
I would suggest using \`\`\`html instead, which is in line with how html code blocks are written in other places in this file.

Reply:
Thanks for checking @nicobako . I picked the syntax above for referencing the script from https://github.com/bigskysoftware/_hyperscript/blob/898345a1753ec365491dd6eedc3ab06873862109/www/docs.md?plain=1#L125. And it renders it like same when building and testing the docs locally.

![Screenshot 2023-12-16 at 14 51 36](https://github.com/bigskysoftware/_hyperscript/assets/106830/9a5c5014-80ca-44ab-862f-dfa17efd96f8)